mne/commands/utils.py

```diff
@@ -4,14 +4,29 @@
 #
 # License: BSD (3-clause)
 
-import imp
+import sys
 import os
 import re
 from optparse import OptionParser
 
 import mne
 
 
+def load_module(name, path):
+    if sys.version_info < (3, 3):
+        import imp
+        return imp.load_source(name, path)
+    elif sys.version_info < (3, 5):
+        from importlib.machinery import SourceFileLoader
+        return SourceFileLoader(name, path).load_module()
+    else:  # Python 3.5 or greater
+        from importlib.util import spec_from_file_location, module_from_spec
+        spec = spec_from_file_location(name, path)
+        mod = module_from_spec(spec)
+        spec.loader.exec_module(mod)
+        return mod
```

Comment:
This is general enough that it should maybe live in `fixes` instead of here. Can you check to see if we use `imp` anywhere else (git grep)?